examples/gallery/embellishments/rose.py

```diff
@@ -0,0 +1,43 @@
+"""
+Rose diagram
+------------
+
+The :meth:`pygmt.Figure.rose` method can plot windrose diagrams or polar histograms.
+"""
+
+import pygmt
+
+data = pygmt.datasets.load_fractures_compilation()
+
+fig = pygmt.Figure()
+
+fig.rose(
+    # use the remote file fractures_06.txt which contains a compilation
+    # of fracture lengths and azimuth as hypothetically digitized from
+    # geological maps
+    length=data.length,
+    azimuth=data.azimuth,
+    # specify the "region" of interest in the (r,azimuth) space [r0, r1, az0, az1],
+    # here, r0 is 0 and r1 is 1, for azimuth, az0 is 0 and az1 is 360 which means
+    # we plot a full circle between 0 and 360 degrees
+    region=[0, 1, 0, 360],
+    # set the diameter of the rose diagram to 7.5 cm
+    diameter="7.5c",
+    # define the sector width in degrees, we append +r here to draw a rose
+    # diagram instead of a sector diagram
+    sector="10r",
```

Comment:
Is this meant to be just `r` or `+r`?

Reply:
It's only ``r``. Adjusted this passage @weiji14 .

Reply:
Actually, it should be **+r** (https://docs.generic-mapping-tools.org/dev/rose.html#a)

Reply:
Hmm, you're right @seisman , however, both version lead to the same result. I used it without the ``+`` since in older versions the example was provided without it (see https://docs.generic-mapping-tools.org/6.0/gallery/ex06.html). Any idea why both versions work? If we decide to use ``+r`` as in 6.2.0 then I have to adjust this also in the docstring.

Reply:
Both `r` and `+r` are supported. `+r` is the new syntax, although the old `r` syntax is still supported.

Reply:
Good to know 😉, will adjust it.